metadata/aquatics-2021-05-01-BTW.yml

```diff
@@ -185,16 +185,16 @@ metadata:
 #match the dimensionality of the initial_conditions unless there are state
 #variables where process error is not being estimated or propagated
     obs_error:
-      status: absent #options: absent, present, data_driven, propagates, assimilates
-      complexity: 2 #Leave blank if status = absent
+      status: data_driven #options: absent, present, data_driven, propagates, assimilates
```

Comment:
will either be `present` or `data_driven` if using the EnKF. I set to `data_driven` because I think the observation uncertainty was guided by reported thermistor uncertainty